gfp_arm64.s

```diff
@@ -105,7 +105,7 @@ TEXT ·gfpMul(SB),0,$0-24
 	MOVD b+16(FP), R0
 	loadBlock(0(R0), R5,R6,R7,R8)
 
-	mul(R9,R10,R11,R12,R13,R14,R15,R16)
+	mul(R9,R10,R11,R12,R13,R14,R15,R16,R17,EMPTY)
```

Comment:
Note: `R17` is used later, but at this point it is uninitialized, so we can use it and not care about it's value being modified.